udata/tests/reuse/test_reuse_model.py

```diff
@@ -10,9 +10,15 @@
 from udata.core.user.factories import UserFactory
 
 from .. import TestCase, DBTestMixin
+from udata.frontend.markdown import init_app
 
 
 class ReuseModelTest(TestCase, DBTestMixin):
+    def create_app(self):
```

Comment:
The impact on circle-ci is more than a minute:

<img width="237" alt="capture d ecran 2017-05-05 a 15 09 13" src="https://cloud.githubusercontent.com/assets/3556/25760475/e32d675a-31a4-11e7-9798-85e9d7f1a2ea.png">